### PR TITLE
pin NCDatasets to v0.12.4

### PR DESCRIPTION
**Pull request by haakon-e**
## Changes

Pin NCDatasets to the release prior to v0.12.5 (most recent release) due to errors described in #401.

## Issue number (if applicable)
Ref: #401.

## Checklist before requesting a review / merging
- [x] I have formatted the code using `julia --project=.dev .dev/climaformat.jl .`
- [x] I have updated all test manifests using `julia --project .dev/up_deps.jl .` with Julia 1.7.3.
- [x] If core features are added, I have added appropriate test coverage.
- [x] If new functions and structs are added, they are documented through docstrings.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -62,7 +62,7 @@ Glob = "1"
 Interpolations = "0.13"
 JLD2 = "0.4"
 JSON = "0.21"
-NCDatasets = "0.11, 0.12"
+NCDatasets = "= 0.12.4"
```
Comment (ilopezgp) [inline Project.toml:65]: why the `=` sign within the quotes?

**Review by ilopezgp — APPROVED** [review]

File: Project.toml (hunk 1 above)
Comment (haakon-e) [reply Project.toml:65]: It pins it to this specific minor release as opposed to allowing future minor releases as well.

Comment (haakon-e) [reply Project.toml:65]: Check this out https://pkgdocs.julialang.org/v1/compatibility/

**Pull request merged**